### PR TITLE
Crypto Utility

### PR DESCRIPTION
**Pull request by weilueluo**
https://jira-eng-rtp3.cisco.com/jira/browse/LH-###### (or Github issue ID)

### CODE REVIEW

**Review by hazital — APPROVED** [review]
LGTM

File: provider/internal/device/ios/resource_test.go [hunk 1]
```diff
@@ -25,8 +25,8 @@ resource "cdo_ios_device" "test" {
 	name = "{{.Name}}"
 	socket_address = "{{.SocketAddress}}"
 	connector_type = "{{.SdcType}}"
-	username = "{{.Username}}"
-	password = "{{.Password}}"
+	username = "{{.EncryptedUsername}}"
```
Comment (bobertyy) [inline provider/internal/device/ios/resource_test.go:28]: The user shouldn't be providing the encrypted username and password here right?

Comment (weilueluo) [reply provider/internal/device/ios/resource_test.go:28]: yes thats correct, hmm this should fail the tests, let me check...

Comment (weilueluo) [reply provider/internal/device/ios/resource_test.go:28]: Fixed CI in https://github.com/CiscoDevNet/terraform-provider-cdo/pull/14

**Pull request merged**